tasks/db_deploy/migrations/migrate_versions_5_to_6/migrate.py

```diff
@@ -5,10 +5,14 @@
 """
 from typing import Dict
 
-from orca_shared.database.shared_db import get_admin_connection, logger
+from aws_lambda_powertools import Logger
+from orca_shared.database.shared_db import get_admin_connection
 
 import migrations.migrate_versions_5_to_6.migrate_sql as sql
 
+# Set AWS powertools logger
+LOGGER = Logger()
```

Comment:
Recommend reverting here, and adding to the card to standardize logger propogation

Reply:
reverted